terratest/src/test/pega/pega-environment-config_test.go

```diff
@@ -29,6 +29,7 @@ func TestPegaEnvironmentConfig(t *testing.T){
 				SetValues: map[string]string{
 					"global.provider":        vendor,
 					"global.actions.execute": operation,
+					"hazelcast.enabled": "true",
```

Comment:
Since hazelcast is like an addon to the main chart, can this test be moved to a new file to isolate with the actual deployments. 
For example, constellation is a new addon to Pega, hence we have added its test cases in a new file. Please refer these two files for reference [1](https://github.com/pegasystems/pega-helm-charts/blob/master/terratest/src/test/pega/pega-tier-deployment-with-c11n_test.go) [2.](https://github.com/pegasystems/pega-helm-charts/blob/master/terratest/src/test/pega/pega-c11n-deployment_test.go)

Reply:
Ok. I will do it. This way it looks clean but we are duplicating the deployments.

Reply:
It doesn't duplicate the code, you can assert only what is required in the new addon's. 
Rest of them will be evaluated as per the actual deployment YAML. Reference links might help you in how to achieve.

Reply:
Yeah , checked it again. We are calling same deployment files for asserting web , batch and stream.